### ISSUE TITLE
Update for Rollup v1

### ISSUE THREAD

**Issue by paulmelnikow** [shrynx/rollup-plugin-cpy#4]
I'm seeing this deprecation message in 1.1.0 when upgrading to Rollup 1.0:

```
(!) rollup-plugin-cpy plugin: The onwrite hook used by plugin rollup-plugin-cpy is deprecated. The generateBundle hook should be used instead.
```

**Comment by paulmelnikow** [comment]
Hey @shrynx, would you consider adding me to this repo and to the npm project so I can get this updated? I'd really appreciate it!

**Comment by shrynx** [comment]
@paulmelnikow Hi paul, sorry for the silence, i haven't been using this package for a while and didn't get notifications from github as well.
anyway i added you as an admin in npm and you can fork the repo, i'll archive it.

**Comment by paulmelnikow** [comment]
Published as 2.0.0! https://www.npmjs.com/package/rollup-plugin-cpy 🎉 

Thanks so much for the access, @shrynx!
